tox.ini

```diff
@@ -68,42 +68,36 @@ whitelist_externals =
 
 [testenv:cypress-dashboard]
 commands =
-    npm install -g npm@'>=6.5.0'
-    pip install -e {toxinidir}/
+    pipenv install --ignore-pipfile
     {toxinidir}/superset/assets/cypress_build.sh dashboard
 setenv =
     PYTHONPATH = {toxinidir}
     SUPERSET_CONFIG = tests.superset_test_config
     SUPERSET_HOME = {envtmpdir}
 deps =
-    -rrequirements.txt
-    -rrequirements-dev.txt
+    pipenv
 
 [testenv:cypress-explore]
 commands =
-    npm install -g npm@'>=6.5.0'
```

Comment:
@harshit-kumar-guavus  As discussed, we should ensure that on dev machine, the mentioned `npm` version is available. We can do that with the help of .nvmrc file and choosing the right node version which ships `npm` version greater than `6.5.0`